### PR TITLE
[ci skip] KDE Gear update to 23.08.4

### PR DESCRIPTION
**Pull request by Luciogi**
#### Testing the changes
- I tested the changes in this PR: YES

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-GLIBC)

## Requires
- [ ] kde Framework 5.113

### CODE REVIEW

**Comment by Luciogi** [comment]
check() is failing:
- [ ] ark
- [ ] kdenlive
- [ ] kdepim-addons
- [ ] kget
- [ ] kgpg
- [ ] kitinerary
- [ ] korganizer

**Comment by Luciogi** [comment]
Remove commits:

- [ ] krdc
- [ ] krfb
- [ ] kwave

**Comment by Johnnynator** [comment]
At least the shib changes of `kate5` are missing

**Comment by Luciogi** [comment]
> At least the shib changes of `kate5` are missing

all these `.so` files points to older version, do these need modification?
```
libKPim5Libkleo.so.5 libkleo-23.04.0_1
libKPim5GrantleeTheme.so.5 grantleetheme-23.04.0_1
libKPim5IMAP.so.5 kimap-23.04.0_1
libKPim5Ldap.so.5 kldap-23.04.0_1
libKPim5AkonadiSearchDebug.so.5 akonadi-search-23.04.0_1
libKPim5AkonadiSearchCore.so.5 akonadi-search-23.04.0_1
libKPim5AkonadiSearchPIM.so.5 akonadi-search-23.04.0_1
libKPim5AkonadiSearchXapian.so.5 akonadi-search-23.04.0_1
libKPim5LibkdepimAkonadi.so.5 libkdepim-23.04.0_1
libKPim5Libkdepim.so.5 libkdepim-23.04.0_1
libKPim5PimCommon.so.5 pimcommon-23.08.0_1
libKPim5PimCommonAkonadi.so.5 pimcommon-23.08.0_1
libKPim5Gravatar.so.5 libgravatar-23.04.0_1
libKPim5SMTP.so.5 ksmtp-23.04.0_1
libKPim5MailTransport.so.5 kmailtransport-23.04.0_1
libKPim5MailTransportAkonadi.so.5 kmailtransport-23.04.0_1
libKPim5Mbox.so.5 kmbox-23.04.0_1
libKPim5MessageComposer.so.5 messagelib-23.04.0_1
libKPim5MessageList.so.5 messagelib-23.04.0_1
libKPim5TemplateParser.so.5 messagelib-23.04.0_1
libKPim5WebEngineViewer.so.5 messagelib-23.04.0_1
libKPim5MessageCore.so.5 messagelib-23.04.0_1
libKPim5MimeTreeParser.so.5 messagelib-23.04.0_1
libKPim5MessageViewer.so.5 messagelib-23.04.0_1
libKPim5CalendarUtils.so.5 kcalutils-23.04.0_1
libKPim5KontactInterface.so.5 kontactinterface-23.04.0_1
libKPim5KSieveUi.so.5 libksieve-23.04.0_1
libKPim5KManageSieve.so.5 libksieve-23.04.0_1
libKPim5KSieve.so.5 libksieve-23.04.0_1
libKPim5MailImporterAkonadi.so.5 mailimporter-23.04.0_1
libKPim5MailImporter.so.5 mailimporter-23.04.0_1
libKPim5MailCommon.so.5 mailcommon-23.04.0_1
libKPim5Tnef.so.5 ktnef-23.04.0_1
libKPim5AkonadiCalendar.so.5 akonadi-calendar-23.04.0_1
libKPim5CalendarSupport.so.5 calendarsupport-23.04.0_1
libKPim5AkonadiNotes.so.5 akonadi-notes-23.04.0_1
libKPim5GAPIPeople.so.5 libkgapi-23.04.0_1
libKPim5GAPIBlogger.so.5 libkgapi-23.04.0_1
libKPim5GAPILatitude.so.5 libkgapi-23.04.0_1
libKPim5GAPICalendar.so.5 libkgapi-23.04.0_1
libKPim5GAPICore.so.5 libkgapi-23.04.0_1
libKPim5GAPITasks.so.5 libkgapi-23.04.0_1
libKPim5GAPIMaps.so.5 libkgapi-23.04.0_1
libKPim5GAPIDrive.so.5 libkgapi-23.04.0_1
libKPim5ContactEditor.so.5 akonadi-contacts-23.04.0_1
libKPim5AkonadiContact.so.5 akonadi-contacts-23.04.0_1
libKPim5AkonadiMime.so.5 akonadi-mime-23.04.0_1
libKPim5Mime.so.5 kmime-23.04.0_1
libkdeconnectinterfaces.so.23 kdeconnect-23.04.0_1
libkdeconnectcore.so.23 kdeconnect-23.04.0_1
libKPim5AkonadiXml.so.5 akonadi5-23.04.0_1
libKPim5AkonadiCore.so.5 akonadi5-23.04.0_1
libKPim5AkonadiPrivate.so.5 akonadi5-23.04.0_1
libKPim5AkonadiWidgets.so.5 akonadi5-23.04.0_1
libKPim5AkonadiAgentBase.so.5 akonadi5-23.04.0_1
libKPim5TextEdit.so.5 kpimtextedit-23.04.0_1
libKPim5IdentityManagement.so.5 kidentitymanagement-23.04.0_1
libKPim5IdentityManagementWidgets.so.5 kidentitymanagement-23.04.0_1
libKPim5EventViews.so.5 eventviews-23.04.0_1
libKPim5IncidenceEditor.so.5 incidenceeditor-23.04.0_1
libKPim5PkPass.so.5 kpkpass-23.04.0_1
libKPim5Itinerary.so.5 kitinerary-23.04.0_1
libKPim5AddressbookImportExport.so.5 kaddressbook-23.04.0_1
libKPim5ImportWizard.so.5 akonadi-import-wizard-23.04.0_1

```

**Comment by Luciogi** [comment]
@Johnnynator if you dont mind, can put kde framework commits in this PR too?
reason: some framework package also depend on devel packages on KDE gear

**Comment by Johnnynator** [comment]
> @Johnnynator if you dont mind, can put kde framework commits in this PR too? reason: some framework package also depend(reverse dependencies) on devel packages on KDE gear

None of these updates (should) need the specific versuib 5.113, the framework update also does not break backwards compat, so no problem with updating it after (or before) updating the apps.

 If you already have the update prepared, you should push it a as a seperate PR.

**Comment by Johnnynator** [comment]
> all these .so files points to older version, do these need modification?

The problem with kate is, that it is a private lib with the specific version in it's name.
All the others are just linked against the generic `.5` soname, so no strict need to update shlibs for them

**Comment by Johnnynator** [comment]
Did you run any cross tests with this? If not I'll give it a test on `aarch64`

File: srcpkgs/kgpg/template [hunk 1]
```diff
@@ -18,4 +20,11 @@ license="GPL-2.0-only"
 homepage="https://kde.org/applications/en/utilities/org.kde.kgpg"
 changelog="https://kde.org/announcements/changelogs/gear/${version}/#kgpg"
 distfiles="${KDE_SITE}/release-service/${version}/src/${pkgname}-${version}.tar.xz"
-checksum=4de7b4b2879f2ae5cbcb3152246e607304aa4dac1da875b1da491908cf9100f7
+checksum=7504ba190ec2385bebd2dea06d0b325cc5fe9a4cb5a5f9a5cff8a961acdfa28b
+
+do_check() {
```
Comment (Johnnynator) [inline srcpkgs/kgpg/template:25]: either overwrite `do_check` or `make_cmd`/`make_check_args`, not both at the same time

**Comment by Luciogi** [comment]
> Did you run any cross tests with this? If not I'll give it a test on `aarch64`

no, I am only testing on x86_64

**Comment by Luciogi** [comment]
I am preparing KDE framework packages, just finalizing failing tests

**Comment by Luciogi** [comment]
If KDE Framework PR is merged first, it would save a compile cycle for KDE gear ;)

**Comment by Johnnynator** [comment]
> If KDE Framework PR is merged first, it would save a compile cycle for KDE gear ;)

You don't need to recompile gear when updating framework. It just works :tm:

File: srcpkgs/kgpg/template (hunk 1 above)
Comment (Luciogi) [reply srcpkgs/kgpg/template:25]: fixed

**Comment by Luciogi** [comment]
kate lib is updated: https://github.com/void-linux/void-packages/pull/47846/commits/4d0727e46e1262fe46ba2846a0244127c04b2428

**Pull request merged**